dd-sdk-android-gradle-plugin/src/test/kotlin/com/datadog/gradle/plugin/internal/GitRepositoryDetectorTest.kt

```diff
@@ -54,7 +53,7 @@ internal class GitRepositoryDetectorTest {
         testedDetector = GitRepositoryDetector()
     }
 
-    @Disabled("This doesn't perform well in the CI, but works locally")
+    // @Disabled("This doesn't perform well in the CI, but works locally")
```

Comment:
??

Reply:
I uncommented it to see what happens on CI (in local everything is fine indeed). Going to pull the image locally and investigate.